etc/shell-logger

```diff
@@ -40,14 +40,14 @@ _LOGGER_DATE="04/Feb/2019"
 LOGGER_DATE_FORMAT=${LOGGER_DATE_FORMAT:-'%Y/%m/%d %H:%M:%S'}
 LOGGER_LEVEL=${LOGGER_LEVEL:-1} # 0: debug, 1: info, 2: notice, 3: warning, 4: error
 LOGGER_STDERR_LEVEL=${LOGGER_STDERR_LEVEL:-4}
-LOGGER_DEBUG_COLOR=${LOGGER_INFO_COLOR:-"3"}
+LOGGER_DEBUG_COLOR=${LOGGER_DEBUG_COLOR:-"3"}
 LOGGER_INFO_COLOR=${LOGGER_INFO_COLOR:-""}
-LOGGER_NOTICE_COLOR=${LOGGER_INFO_COLOR:-"36"}
-LOGGER_WARNING_COLOR=${LOGGER_INFO_COLOR:-"33"}
-LOGGER_ERROR_COLOR=${LOGGER_INFO_COLOR:-"31"}
+LOGGER_NOTICE_COLOR=${LOGGER_NOTICE_COLOR:-"36"}
+LOGGER_WARNING_COLOR=${LOGGER_WARNING_COLOR:-"33"}
+LOGGER_ERROR_COLOR=${LOGGER_ERROR_COLOR:-"31"}
 LOGGER_COLOR=${LOGGER_COLOR:-auto}
 LOGGER_COLORS=("$LOGGER_DEBUG_COLOR" "$LOGGER_INFO_COLOR" "$LOGGER_NOTICE_COLOR" "$LOGGER_WARNING_COLOR" "$LOGGER_ERROR_COLOR")
-if [ "${LOGGER_LEVELS}" = "" ];then
+if [ -z "${LOGGER_LEVELS+x}" ];then
```

Comment:
This might also be better in the form of "${LOGGER_LEVELS:-}",
as `LOGGER_LEVELS` should be an array, not an empty value.
(I think it is much better if it is checked as an array, but it is too heavy here.)